src/Illuminate/Session/Middleware/StartSession.php

```diff
@@ -32,11 +41,13 @@ class StartSession
      * Create a new session middleware.
      *
      * @param  \Illuminate\Session\SessionManager  $manager
+     * @param  \Illuminate\Contracts\Bus\Dispatcher  $dispatcher
      * @return void
      */
-    public function __construct(SessionManager $manager)
+    public function __construct(SessionManager $manager, DispatcherContract $dispatcher)
```

Comment:
as in: `, DispatcherContract $dispatcher = null)`, and update the phpdoc to add `|null`.